lib/colorado_lottery.rb

```diff
@@ -0,0 +1,15 @@
+class ColoradoLottery
+  attr_reader :registered_contestants,
+              :winners
+              :current_contestants
+  def initialize
+    @registered_contestants = {}
+    @winners = []
+    @current_contestants = {}
+  end
+
+  def interested_and_18?(name, lotto)
+    name.age >= 18 && name.game_interest[lotto]
```

Comment:
that renaming would make this part make more sense to a reader (`name.age` doesn't really make sense, but `contestant.age` would)

Reply:
`name.game_interest[lotto]` doesn't work here--  it looks like you're trying to use a key (lotto) to access `name.game_interest` here, but `name.game_interest` is an array, so it doesn't have keys.  something like `name.game_interest.includes?(lotto.name)` would work.